message_sender/tasks.py

```diff
@@ -111,6 +111,13 @@ def run(self, message_id, **kwargs):
                 # send or resend
                 try:
                     if "voice_speech_url" in message.metadata:
+
+                        # OBD number of tries metric
+                        fire_metric.apply_async(kwargs={
+                            "metric_name": 'vumimessage.obd.tries.sum',
+                            "metric_value": 1.0
+                        })
```

Comment:
The way you have it here it will only apply to voice messages. We already have the `vumimessage.tries.sum` on line 141, which should do the same thing, so I think we can leave this metric out.